.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ env:
   PY_COLORS: 1
 
 jobs:
-  test:
+  pytest:
```

Comment:
We'll need to remember to adjust the "required jobs to succeed" setting after merging this I think.

Reply:
I.e. these. So perhaps it would be simpler to just keep it as `test` or WDYT?

![image](https://user-images.githubusercontent.com/630613/209080030-385851bc-3cc6-4dae-8a87-f78830f9c0ad.png)

Reply:
Hmm, where do you find these settings? But sure, that works for me. That also explains why these `test` jobs show up as "required" here..

Reply:
Ah, they need to be changed regardless, because I introduce a new variable.

Reply:
It's under the branch protection settings for the `master` branch: https://github.com/hiboxsystems/marge-bot/settings/branch_protection_rules/32450687

Reply:
Hm, ok. Can we just require the pipeline to be green and not require specific jobs? :thinking:

Reply:
I removed the specific checks now.

Reply:
> Can we just require the pipeline to be green and not require specific jobs? thinking

I'm not sure that works in fact. Try provoking an error now, I think GitHub will still let us merge if we configure it like that. :unamused:

Reply:
![image](https://user-images.githubusercontent.com/1417619/209084024-0f8e02db-cd48-4e53-b7fb-8eb8cb82a34d.png)

Reply:
![image](https://user-images.githubusercontent.com/1417619/209084291-a38f0e8e-fd0d-4535-9959-411e082d4626.png)

Reply:
What insanity is this :grin: But is it a huge problem? It's not green and shiny as it is when all is green, so maybe we can live with it?

Reply:
Insanity it is; not all git providers can apparently be as convenient as GitLab in this regard. :wink: But yes, let's live with it for now. :+1: